### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'assert_hostname'

### ISSUE THREAD

**Issue by VariableDeclared** [kubernetes-client/python#1396]
**What happened (please include outputs or screenshots)**:

When running the `pod_exec.py` example a TypeError is thrown, this appears to come from one of the depencies that needs an arguement updated.

**What you expected to happen**:

The example would run successfully.

**How to reproduce it (as minimally and precisely as possible)**:

```
mkvirtualenv k8s_test_scripts_test --python python2
cd ..
pip install -r requirements.txt
cd examples/
ls
python pod_exec.py
pip install kubernetes
python pod_exec.py
pip install kubernetes==17.14.0a1
python pod_exec.py
```

**Anything else we need to know?**:

Tried on Python3 and Python2

**Environment**:
- Kubernetes version (`kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-20T02:22:41Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.12-gke.1210", GitCommit:"199a41188dc0ca5d6d95b1cc7e8ba96e05f9dd0a", GitTreeState:"clean", BuildDate:"2021-02-05T18:03:16Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}
```
- OS (e.g., MacOS 10.13.6):
Ubuntu 20.04 LTS
- Python version (`python --version`)
`Python 2.7.18`
- Python client version (`pip list | grep kubernetes`)
kubernetes        17.14.0a1

**Comment by Akasurde** [comment]
@VariableDeclared Could you please check #1401 and let me know if it works for you? Thanks.

**Comment by roycaihw** [comment]
/assign @Akasurde 

Fixed by #1401
/close

**Comment by VariableDeclared** [comment]
> @VariableDeclared Could you please check #1401 and let me know if it works for you? Thanks.

That works, thanks!
